nixos/modules/services/web-apps/wordpress.nix

```
@@ -154,6 +158,40 @@ let
           '';
         };
 
+        languages = mkOption {
+          type = types.listOf types.path;
+          default = [];
+          description = lib.mdDoc ''
+            List of path(s) to respective language(s) which are copied from the 'languages' directory.
+
+            ::: {.note}
+            These languags need to be packaged before use, see example.
+            :::
+          '';
+          example = literalExpression ''
+            let
+              # Let's package the German language.
+              # For other languages try to replace language and country code in
+              # the download URL with your desired one. Reference
+              # https://translate.wordpress.org for available translations and
+              # codes.
+              language-de = pkgs.stdenv.mkDerivation {
+                name = "language-de";
+                # Download the language from the Wordpress site. Note the
+                # specific version string which should correspond to the
+                # Wordpress package version you're going to use
+                src = pkgs.fetchurl {
+                  url = "https://de.wordpress.org/wordpress-6.0.1-de_DE.tar.gz";
+                  sha256 = "sha256-dlas0rXTSV4JAl8f/UyMbig57yURRYRhTMtJwF9g8h0=";
+                };
+                # Installing simply means copying all files to the output directory
+                installPhase = "mkdir -p $out; cp -r ./wp-content/languages/* $out/";
+              };
+            # And then pass this language to the languages list like this:
+            in [ language-de ]
```

```suggestion
              })]
```
lets do this without let in to make the example less complex.